static/app/views/onboarding/targetedOnboarding/components/firstEventFooter.tsx

```diff
@@ -39,7 +39,7 @@ export default function FirstEventFooter({
   const getSecondaryCta = () => {
     // if hasn't sent first event, allow skiping.
     // if last, no secondary cta
-    if (!hasFirstEvent || !isLast) {
+    if (!hasFirstEvent && !isLast) {
```

Comment:
@Neo-Zhixing I'm a bit confused about this change. If we're on the last project, why would we ever show the `Next Platform` button?